Instructions.txt

```diff
@@ -0,0 +1,132 @@
+########################################
+### Prepare system (Linux Mint 18.1) ###
+########################################
+
+### Step 1: Create Project Directory
+mkdir TKproj
+cd TKproj
+
+
+### Step 2: Clone sources from github
+git clone https://github.com/kevinhughes27/TensorKart.git
+git clone https://github.com/kevinhughes27/mupen64plus-input-bot.git
+git clone https://github.com/mupen64plus/mupen64plus-core.git
+git clone https://github.com/mupen64plus/mupen64plus-input-sdl.git
+
+
+### Step 3: Install dependancies
+sudo apt-get install dpkg-dev build-essential swig python2.7-dev libwebkitgtk-dev libjpeg-dev libtiff-dev checkinstall ubuntu-restricted-extras freeglut3 freeglut3-dev libgtk2.0-dev  libsdl1.2-dev libgstreamer-plugins-base0.10-dev 
```

Comment:
If you follow this link in the references of that Instructions.txt file, you can see I just took the first step: https://nguyenph88.blogspot.com/2014/04/how-to-install-wxpython.html

I don't know if all those libraries are required, but it (finally) worked...